web/src/ui_init.js

```diff
@@ -623,6 +624,8 @@ export async function initialize_everything(state_data) {
         recent_view_ui.set_initial_message_fetch_status(false);
         recent_view_ui.revive_current_focus();
         server_events.finished_initial_fetch();
+        reactions.initialize_frequently_used_emojis();
+        emoji_picker.initialize();
```

Comment:
I don't think it's safe to not initialize the emoji picker until we have a `message_fetch` response. I think instead, we should just have the emoji picker use the built-in "frequently used" emoji if you open it before we have any data, just like you would in an organization where no messages contain emoji.